src/ui/hash.js

```diff
@@ -119,10 +119,11 @@ class Hash {
     _onHashChange() {
         const loc = this._getCurrentHash();
         if (loc.length >= 3 && !loc.some(v => isNaN(v))) {
+            const bearing = this._map.dragRotate.isEnabled() && this._map.touchZoomRotate.isEnabled() ? +loc[3] : this._map.getBearing() || 0;
```

Comment:
Nit: `getBearing()` always returns a number, so there shouldn't be a need for `|| 0`.